example-context/example-service/example-query/example-query-view/src/main/java/uk/gov/justice/services/example/cakeshop/query/view/service/RecipeService.java

```diff
@@ -28,7 +29,7 @@
      * @return {@link RecipeView} representation of the found recipe, or null.
      */
     public RecipeView findRecipe(final String id) {
-        return recipeRepository.findById(UUID.fromString(id)).map(RecipeView::new).orElse(null);
+        return ofNullable(recipeRepository.findBy(UUID.fromString(id))).map(RecipeView::new).orElse(null);
```

Comment:
Might be more readable if we don't static import Optional.ofNullable and split up to multiple lines.